android-components/components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/activity/GeckoViewActivityContextDelegateTest.kt

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.engine.gecko.activity
+
+import android.app.Activity
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import mozilla.components.support.test.mock
+import org.junit.Assert.assertNull
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.doReturn
+import java.lang.ref.WeakReference
+
+@RunWith(AndroidJUnit4::class)
+class GeckoViewActivityContextDelegateTest {
+
+    @Test
+    fun `getActivityContext returns the same activity as set on the delegate`() {
+        var mockActivity = mock<Activity>()
```

Comment:
All these vars can be vals.

I see this hint in Android Studio, but I guess we don't fail on it, so this is only a nit.